### ISSUE TITLE
Python + XIB steps are hard to follow

### ISSUE THREAD

**Issue by Mark2Mark** [schriftgestalt/GlyphsSDK#64]
I am quite fit with the Interface Builder and Objective-C, but I never used the IB with Python until today.

I tried to follow the steps [here](https://github.com/schriftgestalt/GlyphsSDK/tree/Glyphs3/Python%20Templates#interface-builder-adding-gui-elements), but struggled a lot, because I could not access the outlets from the python file.

I had added the python file, set the File’s owner to it’s main class.
There was no way to ctrl-drag anything from or to the window.

So I ended up editing the XIB as source code and copy-paste 
```xml
<connections>
    <outlet property="fontNameLabel" destination="5LS-iz-okA" id="FTt-lA-0ki"/>
    <outlet property="window" destination="QvC-M9-y7g" id="zeT-oI-P6k"/>
</connections>
```
From an Objecitve-C project and matched the destination IDs.
Am I too stupid, or did things change over time? Has no one ever had any issues? 
Xcode 13.2.1

**Comment by schriftgestalt** [comment]
could you send me the sources that I can give it a try?

**Comment by Mark2Mark** [comment]
Thank you. Preparing another one that fails to connect and will put them both here in a few moment.

**Comment by Mark2Mark** [comment]
The Plugin Maker does not ask for to making a Plugin with or without Dialog, so I fetched one from the "Filter with Dialog" Template and wrapped the view inside a newly added window there.

The Problem, that one cannot drag the connections is independent from if I wrap it in a window or not. It also does not work with the view instead.

When the xib is open in the IB, I add the python file (as described) and set the File’s Owner to the main class.

Now you might not be able to drag any connections, even though you should be. That’s where I was stuck and went for my workaround described above.

I wonder if it has sth to with that I fetched the IBdialog from another template (because the Plugin Maker didn’t offer one). And if so, what’s to change. 

And: what’s the proper way to create the .xib, when one starts with a Template that hasn’t one?

---
Here the plugin which refuses to make connections for you to try:
[Stubborn.glyphsPlugin.zip](https://github.com/schriftgestalt/GlyphsSDK/files/8160712/Stubborn.glyphsPlugin.zip)

And here the one where I manually made the connections in the .xib sourcecode. The Content of that plugin is a but further, and also WIP, so don’t bother with that for now :)

[Plugin With Window.glyphsPlugin.zip](https://github.com/schriftgestalt/GlyphsSDK/files/8160711/Plugin.With.Window.glyphsPlugin.zip)

**Comment by Mark2Mark** [comment]
Submitted an issue at the [pyobjc repo](https://github.com/ronaldoussoren/pyobjc/issues/444). Feel free to chime in or correct my explaination there :)
